src/Mod/Sketcher/Gui/DrawSketchHandlerOffset.h

```diff
@@ -344,6 +344,9 @@ class DrawSketchHandlerOffset: public DrawSketchHandlerOffsetBase
                        std::vector<int>& listOfOffsetGeoIds)
     {
         TopoDS_Shape offsetShape = makeOffsetShape();
+        if (offsetShape.IsNull()) {
+            return;

```

Comment:
Should there be an error of some kind here? This looks like an invisible failure that the user gets no feedback on.